packages/apps/public/locales/en/translation.json

```diff
@@ -1899,4 +1900,4 @@
   "{{value}}x voting balance, locked for {{duration}}x duration{{period}}": "",
   "{{when}} (est.)": "",
   "⚡️ Thunder Gateway": ""
-}
+}
```

Comment:
Guess you've dropped the new line :)

Reply:
it's autogenerated, someone probably tempered with this file before :sweat_smile: